conf/messages

```diff
@@ -1332,18 +1332,21 @@ property-details.multiple-disposals.guidance.p4=You’ll also need to upload a d
 
 enterPostcode.returns.singleDisposal.title=Find the address of the UK property you disposed of
 enterPostcode.returns.agent.singleDisposal.title=Find the address of the UK property your client disposed of
+enterPostcode.returns.personalRepInPeriodOfAdmin.agent.singleDisposal.title=Find the address of the UK property the person disposed of
```

Comment:
does the template return this user key already?

Reply:
Nope, Im still working on that one